packages/paste-website/src/pages/components/anchor/index.mdx

```diff
@@ -51,7 +51,7 @@ export const pageQuery = graphql`
   name={props.pageContext.frontmatter.title}
   categoryRoute={SidebarCategoryRoutes.COMPONENTS}
   githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/anchor"
-  storybookUrl="https://main--5e53448165911c0022e68c74.chromatic.com/?path=/story/components-anchor--default"
+  storybookUrl={`${STORYBOOK_DOMAIN}/?path=/story/components-anchor--default`}
```

Comment:
Oh maybe I wasn't super clear but, you could also do the concat directly in the `ComponentHeader` component to avoid doing it everywhere. No big deal though.

Reply:
that is better still